site/docs-md/basics/configuring-your-app.md

```diff
@@ -117,7 +117,11 @@ The current ones you might configure are:
     // Default is 11.0
     "minVersion": "11.3",
     // Some Cordova plugins require to configure the linker flags
-    "cordovaLinkerFlags": ["-ObjC"]
+    "cordovaLinkerFlags": ["-ObjC"],
+    // Turn off link previews for hyperlinks. `true` by default. Turn this off if your
```

Comment:
I find the turn off and being true default a bit confusing, sounds like it’s off by default, but it’s the opposite.
I think it’s better to stick with the official docs “A Boolean value that determines whether pressing on a link displays a preview of the destination for the link.”
https://developer.apple.com/documentation/webkit/wkwebview/1415000-allowslinkpreview

Reply:
Updated.